.github/workflows/package_installers.yml

```diff
@@ -115,3 +117,33 @@ jobs:
             app/pkg/build/jpackage/*.dmg
             app/pkg/build/jpackage/*.pkg
             app/pkg/build/jpackage/*.deb
+
+      - name: "Upload assets to release"
+        if: github.event_name == 'prerelease' || github.event_name == 'release'
+        env:
+          OS_NAME: '${{ matrix.os }}'
+        uses: actions/github-script@v2
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            console.log(`Running on OS ${process.env.OS_NAME}`);
+            console.log(`Uploading asset to release ID ${context.payload.release.id}`);
+            
+            const os = process.env.OS_NAME.split('-')[0]
+            const fs = require('fs').promises;
+            const { repo: { owner, repo }, sha } = context;
+
+            for (let file of await fs.readdir('./app/pkg/build/jpackage')) {
+              const extension = file.split('.').pop()
+              if (extension != "msi" && extension != "dmg" && extension != "pkg" && extension != "deb")
+                continue;
+            
+              console.log('Uploading', file);
+
+              await github.repos.uploadReleaseAsset({
+                owner, repo,
+                release_id: context.payload.release.id,
+                name: `Installer ${os}.${extension}`,
+                data: await fs.readFile(`./app/pkg/build/jpackage/${file}`)
+              });            
+            }
```

Comment:
We avoid having a scripting code inline by creating a separate script and combining it with `upload-release-asset`

Reply:
Here is an example of a script generating a matrix, https://github.com/MobilityData/gtfs-validator/blob/ec14841a51cecaf7d83d2357cfbe4ac53cd5c660/.github/workflows/acceptance_test.yml#L105

Reply:
Created #1549 to fix that